app/posts/publish-teacher-training-courses/2023-04-20-removing-course-vacancies.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Removing course vacancies
+description: We removed course vacancies as this was misleading for candidates
+date: 2023-04-20
+tags:
+  - school placements
+  - vacancies
+related:
+  items:
+    - text: Removing the school placement vacancy status
+      href: /find-teacher-training/removing-the-school-placement-vacancy-status/
+    - text: Updating course vacancies
+      href: /publish-teacher-training-courses/updating-course-vacancies/
+screenshots:
+  items:
+    - text: Courses list
+      src: courses-list.png
+    - text: Course details with ‘Close course’ button
+      src: course-details--close.png
+    - text: Close course
+      src: close-course.png
+    - text: Course closed
+      src: course-closed.png
+    - text: Course details with ‘Open course’ button
+      src: course-details--open.png
+    - text: Open course
+      src: open-course.png
+    - text: Course opened
+      src: course-opened.png
+    - text: Withdraw open course
+      src: withdraw-open-course.png
+    - text: Withdraw closed course
+      src: withdraw-closed-course.png
+    - text: Training partners’ courses list
+      src: courses-list--training-partners.png
+---
+
+When a provider publishes a course, they can list all the schools where a candidate can gain experience.
+
+Previously, once a course was published, it was open to applications. This meant there were vacancies at some or all of the schools listed.
+
+We provided a way for users to state whether each school had vacancies.
+
+If the provider stated there were no vacancies for all schools on the list, the course is closed.  Closed courses can still be found on Find postgraduate teacher training (Find), but a candidate cannot apply.
+
+We have removed the link between the ‘Open’ and ‘Closed’ course statuses and vacancies. Users can simply state whether the course is open or closed.
+
+## What we changed
+
+We have:
+
+- removed the vacancies section
+- removed the vacancies column from the courses list
+- added closing a course
+- added opening a course
+- updated withdrawing a course
+
+We removed the ability for users to manage vacancies in Publish teacher training courses (Publish) because:
+
+- candidates do not apply to a vacancy at a school
+- providers often do not know the number of vacancies at their partner schools until late in the recruitment cycle
+- the number of vacancies can change during the recruitment cycle for reasons outside the recruitment process
+- we are providing false hope that candidates will be placed at certain schools
+- providers do not always consider a candidate’s school placement preference on the application, and some don’t know about the preference
+- the accuracy of vacancy information depends on providers frequently updating their courses
+- Apply for teacher training (Apply) is going to remove the preference for placement schools
+
+## How it works
+
+When a course is first published, it is ‘open’ on Find.
```

Comment:
Can we stick to one? i.e `‘open’`, `‘Open’` or `open` 🤷🏼

Reply:
> 

Yes, which is why I don't think we need the 'Applications' column. We talked about it in a meeting and I think the overall view was to leave it there for now. I think we should remove in the future though. @victoriawinlow